src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,170 @@
+    public static final Achievement achievementWandOfbuilding = new MineColoniesAchievement("wandofbuilding", "wandofbuilding", 0, -2,
+                                                                                            ModItems.buildTool, achievementGetSupply)
+            .registerStat();
+
+    // Buildings
+    /**
+     * Place a townhall
+     */
+    public static final Achievement achievementBuildingTownhall = new MineColoniesAchievement("townhall", "townhall", -2, 0,
+                                                                                              ModBlocks.blockHutTownHall, achievementGetSupply)
+            .registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingBuilder   = new MineColoniesAchievement("upgrade.builder.first", "upgrade.builder.first", 0, 1,
+                                                                                               ModBlocks.blockHutBuilder, achievementBuildingTownhall)
+            .registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeBuilderMax = new MineColoniesAchievement("upgrade.builder.max", "upgrade.builder.max", 2, 1,
+                                                                                               ModBlocks.blockHutBuilder, achievementBuildingBuilder)
+            .registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingColonist   = new MineColoniesAchievement("upgrade.colonist.first", "upgrade.colonist.first", 0, 2,
+                                                                                                ModBlocks.blockHutCitizen, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeColonistMax = new MineColoniesAchievement("upgrade.colonist.max", "upgrade.colonist.max", 2, 2,
+                                                                                                ModBlocks.blockHutCitizen, achievementBuildingColonist)
+            .registerStat();
+
+    /**
+     * Upgrade a lumberjack to lv 1
+     */
+    public static final Achievement achievementBuildingLumberjack   = new MineColoniesAchievement("upgrade.lumberjack.first", "upgrade.lumberjack.first", 0, 3,
+                                                                                                  ModBlocks.blockHutLumberjack, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a lumberjack
+     */
+    public static final Achievement achievementUpgradeLumberjackMax = new MineColoniesAchievement("upgrade.lumberjack.max", "upgrade.lumberjack.max", 2, 3,
+                                                                                                  ModBlocks.blockHutLumberjack, achievementBuildingLumberjack)
+            .registerStat();
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)